migrations/sqls/20201016094050-allow-billing-and-data-to-edit-charge-workflows-down.sql

```diff
@@ -0,0 +1,9 @@
+-- Remove the agreements role from the groups
+delete from idm.group_roles
+where group_id in (
+  select group_id
+  from idm.groups
+  where groups."group" = 'billing_and_data'
```

Comment:
Although I don't think it causes any issues here, it might be extra-safe to specify 
`and groups.application='water_admin'`